packages/chaire-lib-backend/src/models/db/default.db.queries.ts

```diff
@@ -42,16 +42,27 @@ export const stringifyDataSourceIds = function (dataSourceIds: string[]): string
  * @param id The ID of the object
  * @returns Whether an object with the given ID exists in the table
  */
-export const exists = async (knex: Knex, tableName: string, id: string): Promise<boolean> => {
-    if (!uuidValidate(id)) {
+export const exists = async (
+    knex: Knex,
+    tableName: string,
+    id: string | number,
+    options: {
+        transaction?: Knex.Transaction;
+    } = {}
+): Promise<boolean> => {
+    if (typeof id === 'string' && !uuidValidate(id)) {
```

Comment:
Can you add a comment saying that we now accept both uuids (string) or id (integer)?

Reply:
jsdoc was updated, also with the new options parameters